src/pages/Forms.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import routes from 'constants/routesPaths';
+import {
+  getFormCategories,
+  getFormsByCategory,
+  getFormError,
+  getFormIsFetching
+} from 'selectors/formSelectors';
+import { fetchFormCategories, fetchForms } from 'actions/formActions';
+import Header from 'components/common/Header';
+import Title from 'components/common/Title';
+import Subtitle from 'components/common/Subtitle';
+import Link from 'components/common/Link';
+import Loading from 'components/common/Loading';
+import FormsList from 'components/forms/FormsList';
+import FormsListItem from 'components/forms/FormsListItem';
+
+const ListWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 0;
+  width: 80%;
+  align-self: center;
+  max-width: 730px;
+`;
+
+const Forms = () => {
+  const formCategories = useSelector(getFormCategories);
+  const forms = useSelector(getFormsByCategory);
+  const error = useSelector(getFormError);
+  const isFetching = useSelector(getFormIsFetching);
```

Comment:
These two should be handled using `const { status, error } = useStatus(FETCH_FORMS)` if you have the `statusReducer` from the react-base

Reply:
I will check that